AEPCore/Tests/ClassFinderTests.swift

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+@testable import AEPServices
+@testable import AEPCore
+import XCTest
+
+class ClassFinderTests: XCTestCase {
+
+
+    func testClassFinderSimple() {
+        let foundExtensions = ClassFinder.classes(conformToProtocol: Extension.self)
+        XCTAssertEqual(foundExtensions.count, 13) // All Extensions in AEPCore
```

Comment:
Is this retrieving all extensions from the test target? If so, will it fail if I add a new extension for a specific test? I prefer defining a custom protocol and running tests against it to prevent them from breaking.

Reply:
I think using a custom Protocol here will be fine. I'll make that change.